app/js/script.js

```diff
@@ -0,0 +1,173 @@
+let currentTimeZone = Intl.DateTimeFormat().resolvedOptions().timeZone;
+let manualControlMode = null;
+let intervalId = null;
+let isPaused = false;
+let customTime = null;
+
+const timeZones = [
+  { name: "Local Time", value: currentTimeZone },
```

Comment:
in the js better to use single quotes: { name: 'Local Time', value: currentTimeZone }